tests/handle_unicode.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
```

Comment:
you can get rid of this encoding comment as well

Reply:
Sure, why not.